### DISCUSSION TITLE
Requested VAA not found in store (transfer of WMATIC from POLYGON to ETH stuck)

### DISCUSSION THREAD

**Discussion by Enrik85** [wormhole-foundation/wormhole#2123] [Problems / bugs] [answered]
Hello, 

I can't redeem my tokens using the recovery function in portalbridge. The tx went through: 0x8b01cf6aa7fd47f2537a54da9bb12affa6902ab2b735f0321fa614c278401e6a but when I try to redeem tokens it says "Requested VAA not found in store"

Can anybody help please?

**Comment by kev1n-peters** [comment, marked as answer]
Hello, it looks like this transfer has already been redeemed on ETH. https://etherscan.io/tx/0x2927840e9bd8ea5a33e2ae4f54111b53ea34f5eb1f743f34dbc28931a4c10be9
